.Documentation/articles/developer/10-lockable-properties.md

```diff
@@ -0,0 +1,50 @@
+# Lockable Properties
+
+
+
+## Implementing Lockable Properties
+
+To make your property lockable, inherit from the abstract class called `Innoactive.Creator.Core.Properties.LockableProperty`:
+
+```csharp
+public class LightSwitchProperty : LockableProperty
```

Comment:
Not sure about the Light Switch example. Might give the wrong impression that you should create specific properties for specific objects. This could also be a power switch on a computer, a trap door or whatever. Renaming it just to Switch would be sufficient for me

Reply:
I thought the same, but just "SwitchProperty" might give wrong associations with the `switch` keyword. LeverProperty?